src/toolbar/toolbar-group.jsx

```diff
@@ -197,7 +196,7 @@ const ToolbarGroup = React.createClass({
     }, this);
 
     return (
-      <div {...other} className={className} style={this.prepareStyles(styles.root, style)}>
+      <div {...other} className={className} style={prepareStyles(Object.assign({}, styles.root, style))}>
```

Comment:
However, in this line I think that `prepareStyles(Object.assign(styles.root, style))` would work.